youtube_dl/extractor/abc.py

```diff
@@ -12,7 +12,7 @@
 
 class ABCIE(InfoExtractor):
     IE_NAME = 'abc.net.au'
-    _VALID_URL = r'http://www\.abc\.net\.au/news/[^/]+/[^/]+/(?P<id>\d+)'
+    _VALID_URL = r'http://www\.abc\.net\.au/news/(?:[^/]+/){1,2}(?P<id>\d+)'
```

Comment:
It's not clear in the commit why you are changing this, if you need it you should add an example url in the tests.